tests/test_config_reader.py

```diff
@@ -14,9 +14,9 @@ def test_parse_config_files():
         config_data = parse_config_file("Not_Real_Directory")
         assert pytest_exit_object.type == SystemExit 
         assert pytest_exit_object.value.code == 42  
-        assert pytest_exit_object.sys.stdout.getline().strip() == "Could not find config file. Exiting..."
+        assert pytest_exit_object.sys.stdout.getline().strip() == "Directory containing config file, {}, not found. Exiting...".format("Not_Real_Directory")
     with pytest.raises(SystemExit) as pytest_exit_object:
         config_data = parse_config_file(THIS_DIR)
         assert pytest_exit_object.type == SystemExit 
         assert pytest_exit_object.value.code == 42  
-        assert pytest_exit_object.sys.stdout.getline().strip() == "Could not find directory. Exiting..."
+        assert pytest_exit_object.sys.stdout.getline().strip() == "No file named config.yml found in config directory {}. Exiting...".format(THIS_DIR)
```

Comment:
I think this should be a hard-coded string, not a format string.  i.e., exactly what the string says in the specific case of the failed test.